Makefile

```diff
@@ -75,8 +75,9 @@ site: html pic
 image:
 	docker build -t 'nspccdev/neofs-spec' .
 
-docker/%:
-	docker run --rm -it -v `pwd`:/src -u `stat -c "%u:%g" .` nspccdev/neofs-spec:latest make $$(basename $@)
+docker/pdf:
```

Comment:
This should be a separate change set =)

Reply:
Would you fix it? I'm afraid it will take me much time to deal with it.

Reply:
@aprasolova volunteered to help =)